lib/cc/cli/init.rb

```diff
@@ -65,6 +64,16 @@ def exclude_paths(paths)
         expanded_paths
       end
 
+      def build_engine_config(engine_config)
+        config = { "enabled" => true }
+
+        if engine_config["default_config"]
+          config["config"] = engine_config["default_config"]
+        end
+
+        config
```

Comment:
I think maybe what makes it feel unclean is naming: we're passing around an `engine_config` which is really the "default" or "prototype" engine config, and referring to the actual current engine config as just `config`. (I think: that's my reading of the code.) Agree it's not a blocker, but it does seem it could be made clearer.